// Copyright (C) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

fn main() -> Result<(), Box<dyn std::error::Error>> {
    println!("Tracing example completed successfully!");

    Ok(())
}
